src/jobs/SyncOktaUsersJob.php

```diff
@@ -183,6 +194,10 @@ private function splitUsersIntoCategories(array $users, $uniqueField)
 
         // put users into the category that relates
         foreach ($users as $user) {
+            if (is_array($user)) {
```

Comment:
why do we need to check?  should should pretty consistently be the same variable type?

Reply:
The modification made returns an array but the rest of the functionality expects an object. Safer to cast as object than modify already working functionality.